scripts/install

```diff
@@ -115,10 +112,10 @@ ynh_system_user_create --username=$app
 #=================================================
 # PHP-FPM CONFIGURATION
 #=================================================
-ynh_script_progression --message="Configuring php-fpm..." --weight=2
+ynh_script_progression --message="Configuring php-fpm..." --weight=50
 
 # Create a dedicated php-fpm config
-ynh_add_fpm_config --usage=medium --footprint=high
+ynh_add_fpm_config --usage=medium --footprint=high --phpversion=$YNH_PHP_VERSION --package="$extra_pkg_dependencies"
```

Comment:
If we continue to add the dependencies like that, I think we should rename `extra_pkg_dependencies` into `php_dependencies` or something like that.

Reply:
I totally agree, I thought about it but my computer was already off...
It would be even better as here that's really dependencies for php only.